src/Sinch/Conversation/Contacts/Contact.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Sinch.Conversation.Messages;
+using Sinch.Core;
+
+namespace Sinch.Conversation.Contacts
+{
+    public sealed class Contact
+    {
+        /// <summary>
+        ///     Tracks the fields which where initialized. 
+        /// </summary>
+        private readonly ISet<string> _setFields = new HashSet<string>();
+
+        private List<ChannelIdentity> _channelIdentities;
+        private List<ConversationChannel> _channelPriority;
+        private string _displayName;
+        private string _email;
+        private string _externalId;
+        private string _id;
+        private string _language;
```

Comment:
Yeah, makes sense.
Side comment: did you enjoy working with enums of 20+ values?

Reply:
No 😞 
But from SDK user point of view and because not related to any ISO definition, we can just assuming providing list of supported values will be helpful